spec/rubocop/cop/lint/unified_integer_spec.rb

```diff
@@ -0,0 +1,69 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Lint::UnifiedInteger do
+  let(:config) { RuboCop::Config.new }
+  subject(:cop) { described_class.new(config) }
+
+  %w(Fixnum Bignum).each do |klass|
```

Comment:
I think using shared examples would be better than generating the tests like this.